### PR TITLE
add firebase documentation

### LINKED ISSUE #207: [BONUS]: Format BDD on .md
### Description

Implementation of bdd documentation.

### To do

create a md point containing the firebase documentation.

### Usefulness

useful

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by STom6**
# New feature
creation of a md point containing the firebase documentation.
# Updated feature

# Bug fixed

### CODE REVIEW

**Review by 31Nathan — DISMISSED** [review]
LGTM

**Review by JohanCDev — DISMISSED** [review]
lgtm

File: Server/firebaseDocumentation.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
++ USERS
+	+ uid 1
+		+ AREAS
+			+ uid area 1
+				+ Action
+					+ Description (string)
+					+ serviceName (string)
+					+ subject (string)
+					+ text (string)
+					+ trigger (true / false)
```
Comment (CedricCORGE) [inline Server/firebaseDocumentation.md]: boolean

**Review by CedricCORGE — CHANGES_REQUESTED** [review]

File: Server/firebaseDocumentation.md (hunk 1 above)
Comment (STom6) [reply Server/firebaseDocumentation.md]: done

**Review by CedricCORGE — APPROVED** [review]
lgtm

**Review by JohanCDev — APPROVED** [review]
Lgtm

**Review by osvegn — APPROVED** [review]

**Review by osvegn — APPROVED** [review]

**Pull request merged**